tools/iso/preseed.cfg

```diff
@@ -0,0 +1,1413 @@
+# No partitions to encrypt
+# for internal use; can be preseeded
+d-i	preseed/run/checksum	string	
+# Go back to the menu and correct this problem?
+# Write previous changes to disk and continue?
+# Device file for accessing the CD-ROM:
+d-i	cdrom-detect/cdrom_device	string	/dev/sdb1
+# Network autoconfiguration failed
+netcfg	netcfg/dhcp_failed	note	
+# No boot loader installed
+nobootloader	nobootloader/confirmation_common	note	
+# LILO installation target:
+lilo-installer	lilo-installer/manual_bootdev	string	
+# Enable source repositories in APT?
+apt-setup-udeb	apt-setup/enable-source-repositories	boolean	true
+# Select your time zone:
+# Choices: Eastern, Central, Mountain, Pacific, Alaska, Hawaii, Arizona, East Indiana, Samoa
+tzsetup-udeb	time/zone	select	US/Eastern
+# Error while reducing volume group
+# 
+# Choices: Argentina, Bolivia, Chile, Colombia, Costa Rica, Cuba, Ecuador, El Salvador, Spain, United States, Guatemala, Honduras, Mexico, Nicaragua, Panama, Paraguay, Peru, Puerto Rico, Dominican Republic, Uruguay, Venezuela, other
+d-i	localechooser/shortlist/es	select	
+# state
+# Choices: Acre, Alagoas, Amazonas, Amapá, Bahia, Ceará, Distrito Federal, Espírito Santo, Fernando de Noronha, Goiás, Maranhão, Minas Gerais, Mato Grosso do Sul, Mato Grosso, Pará, Paraíba, Pernambuco, Piauí, Paraná, Rio de Janeiro, Rio Grande do Norte, Rondônia, Roraima, Rio Grande do Sul, Santa Catarina, Sergipe, São Paulo, Tocantins
+tzsetup-udeb	tzsetup/country/BR	select	America/Sao_Paulo
```

Comment:
I really don't know what I'm talking about here, but just wondering what the es, BR, Central America stuff is about.

Reply:
That's because the `debconf-get-selections` program which generated this file originally includes the kitchen sink, including a bunch of comments that don't make a ton of sense. If we want to put additional effort into the installer feature, we might spend a few minutes cleaning up the unnecessary options in this config file.